src/porepy/utils/sort_points.py

```diff
@@ -22,21 +21,21 @@ def sort_point_pairs(
     start end endpoints, so that they form a continuous polyline.
 
     The algorithm is brute-force, using a double for-loop. This can
-    surely be imporved.
+    surely be improved.
 
     Parameters:
     lines: np.ndarray, 2xn, the line pairs. If lines has more than 2 rows, we assume
         that the points are stored in the first two rows.
     check_circular: Verify that the sorted polyline form a circle.
-                    Defaluts to true.
+                    Defaults to true.
     ordering: np.array, return in the original order if a line is flipped or not
```

Comment:
The description of "ordering" is not intuitive to me. Could you improve it if you understand it?

Reply:
The parameter `ordering` has still an effect. If set `True`, a boolean array is returned specifying which of the columns in `lines` had to be flipped in order to construct a chain. If we want to keep this option, I suggest changing the name to something like `return_flip_info` as well as changing the corresponding boolean array `is_ordered` to something like `is_flipped`. On the other hand, this option is actually nowhere used in PP core. 

Any opinion on whether to keep this option? If we want to remove it, we can also remove anything related to `is_ordered`.

Reply:
I think I know where it is used (in an external package of mine). Let's kill it, then I will deal with the consequences.